server/routes/config/curfew.js

```diff
@@ -126,6 +127,57 @@ module.exports = {
       modify: '/hdc/taskList/',
     },
   },
+
```

Comment:
curfewAddressReview and CurfewAddressVersion2 are identical apart from the validation message.

Reply:
Could you just import the version variable into the config and use a ternary to show the validation message based on that version variable?

`validationMessage: curfewAddressVersion === '2' ? 'version 2 message' : 'or original message'`

Reply:
Yes that was my first thought too. But getting the version relies on the licenceService and res object like this
licenceService.getCurfewAddressReviewVersion(res.locals.licence.licence)
And as this is just a config file, I thought it would be okay to not add the extra complexity